src/wizard/python/tracing.md

```diff
@@ -5,7 +5,7 @@ support_level: production
 type: language
 ---
 
-The Sentry SDK for Python does a very good job of auto instrumenting your application. Here a short introduction on how to do custom performance instrumentation. If you'd like to learn more, read our [custom instrumentation](/platforms/python/performance/instrumentation/custom-instrumentation/) documentation.
+The Sentry SDK for Python does a very good job of auto instrumenting your application. Here a short introduction on how to do custom performance instrumentation. If you'd like to learn more, read our [custom instrumentation](https://docs.sentry.io/platforms/python/performance/instrumentation/custom-instrumentation/) documentation.
```

Comment:
Does gatsby not build the link properly without the domain? `/` should have been fine for root, no?

Reply:
Yeah didn't update the python link because it mentions custom instrumentation. 
These updates are for when gap spans pull them like: 
<img width="1433" alt="Screenshot 2023-06-06 at 11 07 59 AM" src="https://github.com/getsentry/sentry-docs/assets/60121741/229a3950-998f-48e7-8855-33d5ee723f2b">